site/src/pages/components/select.mdx

```diff
@@ -0,0 +1,6 @@
+---
+name: Select Element (Explainer)
+layout: ../../layouts/ComponentLayout.astro
+---
+
+See selectlist explainer. [See the explainer here](./selectlist/).
```

Comment:
ideally this (and selectmeny) would just be a redirect but I don't know if that's easy to do till the Astro v3 PR gets merged.